### ISSUE TITLE
Toast mode not showing standard crash alert

### ISSUE THREAD

**Issue by GoogleCodeExporter** [useataaaaa/acra#53]
```
What steps will reproduce the problem?
1. Set up ACRA's toast mode.
2. Write something that will blow.
3. Make it explode.

What is the expected output? What do you see instead?
Android's standard crash alert plus ACRA's toast.

What version of the product are you using? On what operating system?
3.2.1 on android 2.2.1

Please provide any additional information below.
I'm not sure if this is really a bug or just a bad decision.
I think it's very important to maintain the standard android usability across 
applications, otherwise users get confused. I think that the toast notification 
should be a way to add a custom message to the crash but the standard alert 
still should be displayed. This is different with the other mode that allows 
the user to enter a  message because that one is similar to the standard alert 
and it contains a message explaining that the app crashed. But having just a 
toast it's just not enough and inconsistent with the other 99% of apps.

Sorry for the crappy English.
```

Original issue reported on code.google.com by `zeolifra...@gmail.com` on 23 May 2011 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
My own opinion is that users should not be alerted twice for the same issue. 
This adds to the feeling of bad quality. Moreover, for applications distributed 
on the Android Market, this would mean that users will see a first message 
(toast) telling that a report is sent... just before displaying the standard 
"Force Close" dialog with its own "Report" button.

Anyway, I think the real good decision would be to let the developer choose. 
I'll introduce a new ACRA configuration item for this. 
```

Original comment by `kevin.gaudin` on 23 May 2011 at 8:53
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
You are right about the double notification. But I'm also right about standard 
usability across Android...
```

Original comment by `zeolifra...@gmail.com` on 23 May 2011 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
That's why it is now up to the developer to choose.
The following snapshot version contains a new 
@ReportsCrashes.forceCloseDialogAfterToast configuration item.
http://acra.googlecode.com/svn/repository/snapshots/org/acra/acra/4.0.2-SNAPSHOT
/acra-4.0.2-20110523.215410-1.jar

I'll test it tomorrow.
```

Original comment by `kevin.gaudin` on 23 May 2011 at 9:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
My test is ok. Published v4.1.0b with the new feature.
```

Original comment by `kevin.gaudin` on 24 May 2011 at 10:36
- Changed state: **Verified**
- Added labels: **Milestone-Release4.1.0b**
